index.js

```diff
@@ -617,6 +617,15 @@
     function(x) { return RegExp_._test(x) && !x.global; }
   );
 
+  //# NonNegativeInteger :: Type
+  //.
+  //. Type comprising every non-negative Integer value (including -0).
```

Comment:
Let's wrap `Integer` and `-0` in backticks, and have the former link to `#Integer`.